frontend/src/components/content/viewContent.tsx

```
@@ -103,25 +108,36 @@ export default function ViewContent({ id }: ViewContentProps) {
-      .then((res) => {
-        const fetchedContent = res.data;
-        if (fetchedContent.dateCreated && fetchedContent.dateCreated.seconds) {
-          fetchedContent.dateCreated = new Date(
-            fetchedContent.dateCreated.seconds * 1000
-          );
-        } else {
-          fetchedContent.dateCreated = new Date(fetchedContent.dateCreated);
-        }
+    axios.get(`${apiURL}/content/${id}`).then((res) => {
+      const fetchedContent = res.data;
+      if (fetchedContent.dateCreated && fetchedContent.dateCreated.seconds) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)